### ISSUE TITLE
accept ssh option from .csshrc

### ISSUE THREAD

**Issue by GoogleCodeExporter** [brockgr/csshx#28]
```
I added the few lines to the script to accept the "ssh" config option from
.csshrc.  This allows you to designate a different script to use as the
base ssh command.  I use it to put in a helper script that allows me to use
csshX with a jump box.  

Example: create a helper script called "sshgw" like so

#!/bin/bash

jumpbox=JUMPBOX

if [ $4 ] ; then
  host=$4
else
  host=$1
fi


ssh -o ConnectTimeout=10 $jumpbox -t ssh $host

add "ssh=/home/user/sshgw" to your .csshrc file.

Now when you run "csshX host1 host2" it will connect through the JUMPBOX.


```

Original issue reported on code.google.com by `mtbee...@gmail.com` on 20 Apr 2010 at 6:55

Attachments:
- [csshdiff.txt](https://storage.googleapis.com/google-code-attachments/csshx/issue-28/comment-0/csshdiff.txt)

**Comment by GoogleCodeExporter** [comment]
```
This is a feature I always meant to implement but never got around to.

The patch pointed me in the right direction and I added a command line switch 
too.

It's checking into svn (csshX v0.64-110) and I'll roll it out as part of the 
next release.

Thanks!!
```

Original comment by `gavin.brock` on 21 Apr 2010 at 12:49
- Changed state: **Fixed**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Released in 0.65 and 0.70 releases
```

Original comment by `gavin.brock` on 23 Apr 2010 at 4:00
- Changed state: **Done**

**Comment by GoogleCodeExporter** [comment]
```
Sorry - major bug in 0.70, please try 0.71
```

Original comment by `gavin.brock` on 25 Apr 2010 at 2:25
